client/network/src/service.rs

```diff
@@ -889,16 +889,16 @@ impl Metrics {
 			connections_closed_total: register(CounterVec::new(
 				Opts::new(
 					"sub_libp2p_connections_closed_total",
-					"Total number of connections closed, by reason and direction"
+					"Total number of connections closed, by direction, reason and by being the first or not"
 				),
-				&["direction", "reason"]
+				&["direction", "reason", "was_first"]
```

Comment:
```suggestion
				&["direction", "reason", "was_last"]
```

Connections can close in a different order as they were opened.